### PR TITLE
[ikanji]-Sage-how-to-build-a-marketplace-for-selling-shoes-on-the-celo-blockchain

### PR DESCRIPTION
**Pull request by Ikanji201**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+hide_table_of_contents: true
+slug: /tutorials/How-to-build-a-marketplace-for-selling-shoes-on-the-celo-blockchain
+---
+
+
+![header](../../src/data-tutorials/showcase/intermediate/How-to-Build-a-Marketplace-for-Selling-Shoes-on-Celo.png)
+
+## INTRODUCTION
+
+### What is blockchain all about?
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:19]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+Blockchain is a system of recording information in a way that makes it difficult or impossible to alter, hack or cheat the system.
+
+### What is the Celo blockchain?
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:22]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+![header](../../src/data-tutorials/showcase/intermediate/How-to-Build-a-Marketplace-for-Selling-Shoes-on-Celo.png)
+
+## INTRODUCTION
+
+### What is blockchain all about?
+Blockchain is a system of recording information in a way that makes it difficult or impossible to alter, hack or cheat the system.
+
+### What is the Celo blockchain? 
+Celo is a carbon-negative, permissionless, layer-1 blockchain with a rich ecosystem of global partners building innovative Web3 dapps to support a more inclusive financial system. You can read more about Celo by visiting this [page](https://celo.org).
+### What are smart contract?
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:24]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+
+### What is blockchain all about?
+Blockchain is a system of recording information in a way that makes it difficult or impossible to alter, hack or cheat the system.
+
+### What is the Celo blockchain? 
+Celo is a carbon-negative, permissionless, layer-1 blockchain with a rich ecosystem of global partners building innovative Web3 dapps to support a more inclusive financial system. You can read more about Celo by visiting this [page](https://celo.org).
+### What are smart contract? 
+According to Dapp University, smart contracts are where all the business logic of our applications lives. Smart contracts are in charge of reading and writing data to the blockchain, as well as executing business logic. Smart contracts are written in a programming language called Solidity, which looks a lot like Javascript.
+
+# REQUIREMENT
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:27]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 5]
```diff
@@ -0,0 +1,474 @@
...
+- A code editor or text editor, for this tutorial we will be using [Remix](https://remix.ethereum.org/).
+- An Internet Browser and a good internet connection.
+  
+# PREREQUISITE
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:31]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 6]
```diff
@@ -0,0 +1,474 @@
...
+# PREREQUISITE 
+- Basic knowledge of [Javascript](https://www.javascript.com/).
+- Understand how Blockchain works.
+- Have a basic knowledge of [Solidity](https://soliditylang.org/).
+  
+
+ 
+### Now lets begin by creating our smart contract
+
+##  Who this course is for:
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:40]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 7]
```diff
@@ -0,0 +1,474 @@
...
+- Anyone who wants to get started with smart contracts.
+- Take this tutorial if you want to get a clear understanding of how the Celo blockchain works.
+ 
+## What We’ll Be Building
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:44]: One line space after each heading and subheading

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 8]
```diff
@@ -0,0 +1,474 @@
...
+
+The Pragma is used to specify what version of Solidity our smart contracts use and thereby help the compiler to decide on the required.
+
+**How do you know which version of Solidity to use?** <br>
+It's always good to use the latest version of solidity except if you have some limiting factors.
+Pragma solidity >=0.7.0 <0.9.0: This means that our smart contract code is to be compiled with a version of Solidity that is greater than or equal to 0.7.0 but less than 0.9.0.
+
+Next, we would be discussing the IERC20 token which enables us to carry out transactions with the Celo Usd stable coin (cUSD).
+
+## What is ERC20?
```
Comment (Ishanpathak1) [inline blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:68]: One line space after each heading and subheading

**Review by Ishanpathak1 — COMMENTED** [review]
Hey, remove the <br> from the .md file and leave one line space after each heading, and also it would be great ig you can provide the git repo, if it has any

**Review by Ishanpathak1 — COMMENTED** [review]
Make the changes and let me know

File: blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md [hunk 9]
```diff
@@ -0,0 +1,494 @@
...
+            msg.sender == shoes[_index].owner,
+            "Only the shoe owner can update the shoe's details"
+        );
+        Shoe storage currentShoe = shoes[_index];
+        currentShoe.image = _newImage;
+        currentShoe.brand = _newBrand;
+    }
+}
+ ```
+
```
[the remark this answers is not in the document]
Comment (Ikanji201) [reply blog/02-18-2023-How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain/How-to-Build-a-Marketplace-For-Selling-Shoes-on-The-Celo-Blockchain.md:443]: HI @Ishanpathak1 this tutorial only just covers the smart contract only.

**Review by viral-sangani — APPROVED** [review]

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**